### PR TITLE
Support hangup verb

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review

File: vumi_twilio_api/twilio_api.py [hunk 1]
```diff
@@ -218,7 +213,18 @@ def new_session(self, message):
...
-            yield self._handle_twiml_verb_reply(verb, message)
+            if verb.name == "Play":
+                yield self.reply_to(message, None, helper_metadata={
+                    'voice': {
+                        'speech_url': verb.nouns[0],
+                        }
+                    })
+            elif verb.name == "Hangup":
+                yield self.reply_to(
+                    message, '',
```
Comment (hodgestar) [inline vumi_twilio_api/twilio_api.py]: I think we should set content to `None` here too?

**Comment by hodgestar** [comment]
Left one very minor comment. Otherwise looks good.

File: vumi_twilio_api/twilio_api.py (hunk 1 above)
Comment (rudigiesler) [inline vumi_twilio_api/twilio_api.py]: Missed that one. Thanks

**Comment by rudigiesler** [comment]
Issues addressed.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**